src/components/ReportActionItem/TaskAction.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {View} from 'react-native';
+import Text from '@components/Text';
+import * as TaskUtils from '@libs/TaskUtils';
+import useThemeStyles from '@styles/useThemeStyles';
+
+type TaskActionProps = {
+    /** Name of the reportAction action */
+    actionName: string;
```

Comment:
i notice we are passing a value to `taskReportID` for this component, should we add it in this prop type?

Reply:
@getusha I think `taskReportID` is not used anymore in this component

Reply:
@VickyStash i noticed this comment above `taskReportID` in proptypes
`// eslint-disable-next-line react/no-unused-prop-types -- This is used in the withOnyx HOC`, so you're saying it is not used anymore in withOnyx HOC?

Reply:
@getusha This component doesn't have withOnyx HOC where `taskReportID` was used, it was [removed](https://github.com/Expensify/App/pull/27119/commits/9780eb9162cca57fa602380139cf7e1146e833ab#diff-9ad37d9be0f6d0266256623fda924ea8a11c7d75aab6711e0c2347bd524ec2d6L56) some time ago.

Reply:
After some time `taskReportID ` was also used in another place, but it also [was updated](https://github.com/Expensify/App/pull/32249/files#diff-9ad37d9be0f6d0266256623fda924ea8a11c7d75aab6711e0c2347bd524ec2d6L25)

Reply:
Could we remove this if that's the case, it is kind of confusing.
https://github.com/Expensify/App/blob/7f7cfb318bdb073cbaf478843255c2d5f9423d12/src/pages/home/report/ReportActionItem.js#L363

Reply:
@getusha Done!